king_phisher/client/tabs/mail.py

```diff
@@ -68,6 +68,14 @@
 else:
 	import urllib.parse
 
+try:
+	from gi.repository import WebKit2 as WebKitX
+	has_webkit2 = True
+except ImportError:
+	from gi.repository import WebKit as WebkitX
+	has_webkit2 = False
+
+
```

Comment:
Infact this entire section does not seem necessary since webkit is never referenced.